app/boards/arm/corneish_zen/corneish_zen_v2_left.keymap

```diff
@@ -0,0 +1,8 @@
+/*
+*
+* Copyright (c) 2021 Darryl deHaan
+* SPDX-License-Identifier: MIT
+*
+*/
+
+#include "corneish_zen.keymap"
```

Comment:
I believe the left/right keymap files are unneeded now after some build fixes. Can you please test and if so, unify this?